src/pipeline-server/Dockerfile.pipeline-server

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-FROM intel/dlstreamer-pipeline-server:2022.2.0-ubuntu20-gpu419.40
+FROM intel/edge_video_analytics_microservice:2.0.0
```

Comment:
I didn't find this public image on dockerhub yet. How are we supposed to download it ? https://hub.docker.com/r/intel/edge_video_analytics_microservice/tags